lametro/events.py

```diff
@@ -132,7 +132,14 @@ def _merge_events(self, events):
                 try:
                     assert event.key not in spanish_events
                 except AssertionError:
-                    raise AssertionError('{0} already exists as a key with a value of {1}'.format(event.key, spanish_events[event.key]))
+                    # Don't allow SAP events to be overwritten in the event
```

Comment:
@fgregg We separate incoming events into an array of English events and a dictionary of Spanish events keyed on the event body name and event date. Because it's a dictionary, i.e., keys are unique, more than one match isn't possible, but keys can be overwritten if our matching criteria is too broad. We already had an assertion in place to prevent overwrites. I added a comment to clarify why this is important.